clients/src/test/java/org/apache/kafka/common/network/SelectorTest.java

```diff
@@ -111,7 +111,7 @@ public void testCantSendWithoutConnecting() throws Exception {
      */
     @Test(expected = IOException.class)
     public void testNoRouteToHost() throws Exception {
-        selector.connect("0", new InetSocketAddress("asdf.asdf.dsc", server.port), BUFFER_SIZE, BUFFER_SIZE);
+        selector.connect("0", new InetSocketAddress("asdf.asdf.dsc.local", server.port), BUFFER_SIZE, BUFFER_SIZE);
```

Comment:
Good point. Just a minor suggestion: should we change this to use "some.invalid.hostname.foo.bar.local" as well?